gcc/config/riscv/riscv.opt

```diff
@@ -71,9 +71,21 @@ Target Report Mask(ATOMIC)
 Use hardware atomic memory instructions.
 
 mmuldiv
-Target Report Mask(MULDIV)
+Target Report RejectNegative Var(TARGET_muldiv)
 Use hardware instructions for integer multiplication and division.
 
+mno-muldiv
+Target Report RejectNegative Undocumented Var(TARGET_no_muldiv)
+Dont Use hardware instructions for integer multiplication and division.
```

Comment:
I think let remove  RejectNegative for mmuldiv is more reasonable since mno-muldiv is Undocumented, 
and Var(TARGET_muldiv) can remove if it's unused.